src/builtin.c

```diff
@@ -1390,10 +1409,10 @@ static block bind_bytecoded_builtins(block b) {
                                             BLOCK(gen_param("start"), gen_param("end")),
                                             range));
   }
-
-  return block_bind_referenced(builtins, b, OP_IS_CALL_PSEUDO);
+  return block_bind(builtins, b, OP_IS_CALL_PSEUDO);
```

Comment:
So, this means we don't lose the bytecode for unreferenced builtins.  That seems... sad?  :)

Is there some other way?